lib/bloc/user_profile/user_profile_bloc.dart

```diff
@@ -329,6 +333,14 @@ class UserProfileBloc {
     });
   }
 
+  _listenPosCurrencyChanges(ServiceInjector injector) {
```

Comment:
Instead of introducing another sink, isn't it better to use an AsyncAction?

Reply:
[Done](https://github.com/breez/breezmobile/pull/309/commits/a01c04f3e980c13d1ddf5ae97b9f99fd76f70c13)